src/Spago/PackageSet.hs

```diff
@@ -40,6 +42,100 @@ makePackageSetFile force comments = do
     else logWarn $ display $ Messages.foundExistingProject packagesPath
   Dhall.format packagesPath
 
+-- | Use the specified version of the package set (if specified).
+--   Otherwise, get the latest version of the package set if possible
+modifyPackageSetVersion
+  :: forall env
+  .  (HasLogFunc env, HasGlobalCache env)
+  => Maybe Text
+  -> RIO env ()
+modifyPackageSetVersion = maybe upgradePackageSet useSpecificRelease
```

Comment:
```suggestion
updatePackageSetVersion
  :: forall env
  .  (HasLogFunc env, HasGlobalCache env)
  => Maybe Text
  -> RIO env ()
updatePackageSetVersion = maybe upgradePackageSet useSpecificRelease
```

This is a minor nit, but I feel "update" fits better what we're doing here

Reply:
I've updated the code to use that name, but I don't think one will always use it to "upgrade" the package set. One could also use `spago upgrad-set --tag olderVersion` to downgrade the package set to an earlier one.